adsmp/app.py

```diff
@@ -108,6 +108,27 @@ def update_storage(self, bibcode, type, payload):
                 oldval = 'not-stored'
                 r.metrics = payload
                 r.metrics_updated = now
+            elif type == 'augment':
```

Comment:
`augment_records` is missing here - which is also something I'd like to address

the current pipeline is working fine with the individual updates; ie. sending around messages tied to one bibcode; from which comes my main worry - the code would be much nicer if we didn't have to deal with the list of `AugmentationRecords` (frankly, those are just repeated `AugmentationRecord`)

can we do that? rephrased: why do we need the `record list`


btw: sending a list of `AugmentAffiliationResponseRecord` would work too; but obviously be breaking our own rule of always using a protobuf intra-pipeline communication

Reply:
I don't think augments_records isn't need there.  The task that receives the list of augments iterates over them and updates the storage individually.  https://github.com/spacemansteve/ADSMasterPipeline/blob/110e68ab02d8c14f727e19e31776a21d51013b88/adsmp/tasks.py#L75  

Regarding the lists of records, should I create an issue and delete them from all pipelines?  Do you want that in this sprint or a future one?